lsp-mode.el

```diff
@@ -7031,6 +7032,12 @@ SESSION is the active session."
   :package-version '(lsp-mode . "6.3")
   :group 'lsp-mode)
 
+(defcustom lsp-verify-signature t
```

Comment:
Should it be enabled by default? Not sure how gpg works on general user setup, e.g. mac.

Reply:
You may create a test - our test suite runs on both mac and win.

Reply:
I doubt it will be indicative - It will require gnupg to be installed, so either I'll need to install it or it won't pass :)

Reply:
I'd suppose that most linux users will have it, but mac users - won't.

Reply:
Hm, but I can just check it :)

Reply:
Changed its value to `(executable-find epg-gpg-program)` - should solve most of the cases, I think.

Reply:
I guess you can do executable-find and in case it is not present print warning/info message that the validation will be skipped and continue. Eventually, you can fail the installation and ask either to disable the validation or to install the needed binaries.

Reply:
ok, changed to a warning.